pkg/instrumentation/sdk.go

```diff
@@ -30,6 +35,24 @@ const (
 	volumeName        = "opentelemetry-auto-instrumentation"
 	initContainerName = "opentelemetry-auto-instrumentation"
 
+	resourceK8sNsName          = "k8s.namespace.name"
```

Comment:
Just to confirm, this is in line with the semantic conventions spec, right? Perhaps it would be a good idea to add a link to it?
https://github.com/open-telemetry/opentelemetry-specification/blob/v1.8.0/specification/resource/semantic_conventions/k8s.md

Reply:
Any thoughts on importing the SDK definitions?

https://github.com/open-telemetry/opentelemetry-go/blob/main/semconv/v1.7.0/resource.go#L502

While they may look like a fancy type, they're just strings

https://github.com/open-telemetry/opentelemetry-go/blob/be0d9c6ea1192bb164d84202e3c23d216619de3b/attribute/key.go#L19

Perhaps a reason not to is since it would bring in OTel - but I'm guessing we do want to instrument our webhook with OTel at least someday :)

Reply:
if it does not pull in a lot of unnecessary deps I am fine with it.

Reply:
it seems to be fine, I have changed it to use the contents from OTEL client